aries-site/src/examples/templates/dashboards/content/chartCards/CostCompact.js

```diff
@@ -0,0 +1,181 @@
+import { useContext, useEffect, useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Grid,
+  Meter,
+  ResponsiveContext,
+  Stack,
+  ThemeContext,
+} from 'grommet';
+import { parseMetricToNum } from 'grommet/utils';
+import { ChartCard, Legend, Measure } from '../../components';
+import { defaultWindow, formatCurrency, REPORT_WINDOW_MAP } from './utils';
+
+const MOCK_DATA = require('../../../../../data/mockData/consumption.json');
+
+export const CostCompact = ({ period }) => {
```

Comment:
Since this is variant presentation of the CostByService, would it make sense to name them so that they appear together in the directory? Open to names, but would like them to share something in common so they are easily identifiable as being related.

```suggestion
export const CostByServiceCompact = ({ period }) => {
```

Reply:
I like this, will update.